react-frontend/src/components/Home/cardList.js

```diff
@@ -76,13 +79,9 @@ const CardList = () => {
             />
           </Col>
         </Row>
-        <Row style={{ paddingTop: '30px' }}>
-          <Col sm={12}>
-            <Title2>Learn from Others</Title2>
-          </Col>
-        </Row>
+
         <Row>
-          <Col sm={12} md={6}>
+          <Col sm={12} md={4}>
```

Comment:
Changing this to 4 means it's going to try putting 3 columns  of cards.  It makes more sense to add a new row to hold the last card,

Reply:
Thanks for that catch—I was too hasty there. Resolved with this commit: https://github.com/bcgov/digital.gov.bc.ca/pull/855/commits/bd7b47163939bb642ab1fb8a1b8045aa034cc1cb